### PR TITLE
A-1203713207662199

### PR DESCRIPTION
**Pull request by owlsua**
# Description

Add entropy-generator library, connect the drawing data with the entropy generation

# :clipboard: Checklist:

- [x] I have named my branch as `A-[id of Asana task]`
- [x] I have set the title of my PR as `A-[id of Asana task]: [short description]`
- [x] My changes passed successfully by `prettier` check
- [x] My changes passed successfully by `lint` check
- [x] My changes kept the previous test coverage rate
- [x] I have added enough tests for my new feature/bugfix
- [x] I have set at least one person to review this PR
- [x] I have set myself as the assignee of ths PR
- [x] I have set at least one label to this PR

### CODE REVIEW

File: src/utils/Helpers/Array/Array.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const getNRandomElementsFromArray = (array, n) => {
```
Comment (renie) [inline src/utils/Helpers/Array/Array.js:1]: Can we please have a test for this?

**Review by renie — CHANGES_REQUESTED** [review]
The restoring process progress steps do not look right:
Screenshot from 2023-03-01 16-23-24

It should look like on creation screen, where the bars almost touch each other:
Screenshot from 2023-03-01 16-23-31

File: src/utils/Helpers/Array/Array.js (hunk 1 above)
Comment (owlsua) [reply src/utils/Helpers/Array/Array.js:1]: Done

**Review by renie — APPROVED** [review]

**Comment by renie** [comment]
Sound good to me. Just change the target branch to `dev` before merging, then merge it, and close PR #73 with a comment pointing to this branch, please.

**Pull request merged**